code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -157,6 +182,29 @@ type StorageChallenge struct {
 	Response       *ChallengeResponse `json:"challenge_response,omitempty"`
 }
 
+func (sc *StorageChallenge) isChallengePassed(success, failure int,
+	challengeCompletionTime time.Duration, now common.Timestamp) (pass, fresh bool, threshold int) {
+
+	cct := toSeconds(challengeCompletionTime)
+	threshold = len(sc.Validators) / 2
+	pass = success > threshold ||
+		(success > failure && success+failure < threshold)
+	fresh = sc.Created+cct >= now
```

Comment:
We should always require success > threshold

Reply:
@peterlimg this was existing code, which I moved to a function as this was needed earlier.
reverting it back to older state